### PR TITLE
Add replace and rename subcommand for !notes

### LINKED ISSUE #156: Add !note rename {note} {newname}
Renaming notes would be a nice feature to have

### LINKED ISSUE #169: On Note Add, it checks for existing notes BEFORE forcing it to lower-case.
This means you can do
image

### LINKED ISSUE #170: `list` is considered a banned word, so users are unable to do !note list
Solution; remove list as a keyword for the command. Since !note has the same effect.

### PR DESCRIPTION
**Pull request by stelar7**
Fixes #156 
Fixes #170 
Fixes #169

### CODE REVIEW

File: src/Info.ts [hunk 1]
```diff
@@ -211,6 +211,51 @@ export default class Info {
...
+            const info = this.infos.find(inf => {
+                return inf.command === args[1];
+            });
+
+            if (info) {
+                info.command = args[2];
+                message.channel.send("new name is now " + info.command);
+            } else {
+                message.channel.send("no valid command");
+            }
```
Comment (molenzwiebel) [inline src/Info.ts]: Consider some capitalization and proper sentences.

File: src/Info.ts [hunk 2]
```diff
@@ -211,6 +211,51 @@ export default class Info {
...
+                return inf.command === args[1];
+            });
+
+            if (info) {
+                const body = args.splice(2).join(" ");
+                info.message = body;
+                message.channel.send("new body is now " + body);
+            } else {
+                message.channel.send("no valid command");
+            }
```
Comment (molenzwiebel) [inline src/Info.ts]: Same here

**Comment by Querijn** [comment]
What happens if I rename a note to another note's command?

**Comment by Querijn** [comment]
Also consider making readable branch-names

**Comment by Querijn** [comment]
Could you comment on this?

**Comment by stelar7** [comment]
Ill make an update to this once i get home today

**Comment by Querijn** [comment]
Those are case-sensitive compares, you gotta make sure that the handleNoteRename has a lowercase names for both old and new name

**Pull request merged**